editor/src/core/data-tracing/data-tracing.ts

```diff
@@ -436,6 +440,87 @@ function walkUpInnerScopesUntilReachingComponent(
   )
 }
 
+const hookCallRegex = /^(use[A-Za-z]+)\(/
```

Comment:
in theory this could also contain digits and underscores, right?

`use[a-zA-Z0-9_]+`

Reply:
Yes, in fact I think I can go quite wide on this because it's guaranteed to be an identifier, maybe.

Reply:
Improved in 5c8e5d0db.